internal/uvm/create.go

```diff
@@ -160,6 +163,9 @@ func verifyOptions(ctx context.Context, options interface{}) error {
 		if len(opts.LayerFolders) < 2 {
 			return errors.New("at least 2 LayerFolders must be supplied")
 		}
+		if opts.SCSIControllerCount < 1 {
+			return errors.New("at least 1 SCSI controller is required for WCOW")
```

Comment:
Can we do this check in `CreateWCOW` if it's only needed for WCOW?

Reply:
@kevpar  PTAL.

Reply:
I guess it's fine to have here since it's under the WCOW case block.